charts/argo-rollouts/templates/controller/hpa.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.controller.autoscaling.enabled }}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "argo-rollouts.fullname" . }}
+  namespace: {{ include  "argo-rollouts.namespace" . }}
```

Comment:
Since `.Release.Namespace` is used in many places, please update other ones as well.

Reply:
I updated this line, thanks! :)